op-node/chaincfg/chains.go

```diff
@@ -196,13 +200,15 @@ var OPBNBDevnet = rollup.Config{
 	MaxSequencerDrift:      600,
 	SeqWindowSize:          14400,
 	ChannelTimeout:         1200,
-	L1ChainID:              big.NewInt(797),
-	L2ChainID:              big.NewInt(1320),
-	BatchInboxAddress:      common.HexToAddress("0xff00000000000000000000000000000000000204"),
-	DepositContractAddress: common.HexToAddress("0xd93160096c5b65bb036b3269eb02328ddadb9856"),
-	L1SystemConfigAddress:  common.HexToAddress("0xf053067cec8d8990de2ba9e17ec2f16c63c7bec4"),
+	L1ChainID:              big.NewInt(714),
+	L2ChainID:              big.NewInt(1322),
+	BatchInboxAddress:      common.HexToAddress("0xff00000000000000000000000000000000001322"),
+	DepositContractAddress: common.HexToAddress("0xb7cdbce0b1f153b4cb2acc36aeb4d9d2cdda1132"),
+	L1SystemConfigAddress:  common.HexToAddress("0x6a2607255801095b23256a341b24d31275fe2438"),
 	RegolithTime:           u64Ptr(0),
-	Fermat:                 big.NewInt(3615117),
+	// Fermat:                 big.NewInt(3615117),
```

Comment:
should this line remain?

Reply:
Currently, `Fermat` is not enabled in our QA environment, so we have removed it for the time being. Can open it together when testing the `Snow` fork. Because the QA environment may be reset frequently, it is more convenient to choose to use a configuration file.